bandit/core/manager.py

```diff
@@ -409,3 +414,63 @@ def _find_candidate_matches(unmatched_issues, results_list):
                                         unmatched == i])
 
     return issue_candidates
+
+
+def _get_tests_from_nosec_comment(comment):
+    nosec_no_space = '#nosec'
+    nosec_space = '# nosec'
+    nospace = comment.find(nosec_no_space)
+    space = comment.find(nosec_space)
```

Comment:
I had to change this slightly, we want to be able to parse out a nosec comment anywhere in the line not just the beginning so I dropped the `^` and switched to `.search` instead of `.match`